src/platforms/common/configuration/options.mdx

```diff
@@ -591,6 +591,15 @@ An optional property that configures which downstream services receive the `sent
 
 <PlatformSection supported={["react-native", "javascript.cordova", "javascript.capacitor", "flutter"]}>
 
+## Experimental Features
```

Comment:
This is the new/common docs for experimental features.
It's empty because no SDKs have it yet.